community/examples/eda/README.md

```diff
@@ -0,0 +1,245 @@
+
+   ```bash
+   ./gcluster create community/examples/eda/<blueprint>.yaml --vars "project_id=${GOOGLE_CLOUD_PROJECT}" --vars region=us-central1 --vars zone=us-central1-a
+   ```
+
+1. Deploy the `base` group
+
+   Call the following gcluster command to deploy the blueprint.
+
+   ```bash
+   ./gcluster deploy <blueprint>
+   ```
+
+   The next `gcluster` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `base` group. Select 'apply'.
+
+   This group will create a network and file systems to be used by the cluster.
+
+   > [!WARNING]
+   > This gcluster command will run through 2 deployment groups (3 if you populate
+   > & activate the `software_installation` stage) and prompt you to apply each one.
+   > If the command is cancelled or exited by accident before finishing, it can
+   > be rerun to continue deploying the blueprint.
+
+1. Deploy the `software_installation` group (_optional_).
+
+   > [!NOTE]
+   > Installation processes differ between applications. Some come as a
+   > precompiled binary with all dependencies included, others may need to
+   > be built from source, while others can be deployed through package
+   > managers such as spack. This deployment group is intended to be used
+   > if the software installation process requires substantial amount of time (e.g.
+   > compilation from source). By building the software in a separate
+   > deployment group, this process can be done before the cluster is
+   > up, minimizing costs.
+   >
+   > [!NOTE]
+   > By default, this deployment group is disabled in the reference design. See
+   > [Software Installation Patterns](#software-installation-patterns) for more information.
+
+   If this deployment group is used (needs to be uncommented in the blueprint first),
+   you can return to the gcluster command which will ask you to **display**, **apply**,
+   **stop**, or **continue** without applying the `software_installation` group.
+   Select 'apply'.
+
+1. Deploy the `cluster` group
+
+   The next `gcluster` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `cluster` group. Select 'apply'.
+
+   This deployment group contains the Slurm cluster and compute partitions.
+
+## Teardown Instructions
+
+> [!NOTE]
+> If you created a new project for testing of the EDA solution, the easiest way to
+> eliminate billing is to delete the project.
+
+When you would like to tear down the deployment, each stage must be destroyed.
+Since the `software_installation` and `cluster` depend on the network deployed
+in the `base` stage, they must be destroyed first. You can use the following
+commands to destroy the deployment in this reverse order. You will be prompted
+to confirm the deletion of each stage.
+
+```bash
+./gcluster destroy <blueprint>
```

Comment:
destroy takes in the deployment folder generated by deploy command.

destroy doesn't understand blueprint, refer - https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/gke-g4/README.md#clean-up